pages/[...slug].vue

```diff
@@ -18,6 +18,39 @@
   <EpPageMeta :doc="doc" />
   <NuxtLayout>
     <div class="px-4 print:px-8 mt-10 mb-96 print:mb-0">
+      <div>
+        <nav>
+          <span v-if="breadcrumbs.length != 1">
+            <NuxtLink :to="'/'">
+              <Icon name="heroicons:home-20-solid" size="2em" />
+            </NuxtLink>
+            {{ '>' }}
```

Comment:
You don't need to past this as JS value.

```suggestion
            >
```

Moreover, it is better to some icon here: https://icones.js.org/collection/heroicons?s=chevron

Reply:
used the chevron right